internal/controller/workspace/workspace_test.go

```diff
@@ -102,10 +104,13 @@ func TestConnect(t *testing.T) {
 	errBoom := errors.New("boom")
 	uid := types.UID("no-you-id")
 	tfCreds := "credentials"
+	zl := zap.New(zap.UseDevMode(true))
+	log := logging.NewLogrLogger(zl.WithName("test"))
```

Comment:
I'd prefer not to enable the logger while running tests. It should be possible to plug in a `logger.NewNopLogger` (iirc) to use a no-op logger.